microarray/src/org/labkey/api/study/assay/matrix/AbstractMatrixRunCreator.java

```diff
@@ -126,12 +123,12 @@ private Map.Entry<DomainProperty, String> findSetProperty(Map<DomainProperty, St
     }
 
     @Override
-    protected void addInputMaterials(AssayRunUploadContext<ProviderType> context,
-                                     Map<ExpMaterial, String> inputMaterials,
-                                     ParticipantVisitResolverType resolverType,
-                                     @NotNull RemapCache cache,
-                                     @NotNull Map<Integer, ExpMaterial> materialCache)
-            throws ExperimentException
+    protected void addInputMaterials(
```

Comment:
These changes ended up being a no-op after further refactoring but I've already got the branch so going to go ahead with integrating.